types/global/managed_config.d.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-type ManagedConfig = {
+type ManagedConfig = JSX.IntrinsicAttributes & {
```

Comment:
This silence the error, but the typing is wrong in emm. Fixed here: https://github.com/mattermost/react-native-emm/pull/9

Once that is fixed, this change is not needed.

Reply:
alright, all this is now done. Thanks @larkox for the help